edn/src/edn.rustpeg

```diff
@@ -148,11 +148,11 @@ namespace_separator = "/"
 // TODO: Be more picky here
 // Keywords follow the rules of symbols, except they can (and must) begin with :
 // e.g. :fred or :my/fred. See https://github.com/edn-format/edn#keywords
-symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
-symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
+symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>.]
+symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>.]
 
-symbol_namespace = symbol_char_initial symbol_char_subsequent* (namespace_divider symbol_char_subsequent+)*
-symbol_name = ( symbol_char_initial+ / "." ) ( symbol_char_subsequent* / "." )
```

Comment:
Correct, `...` would be a `PlainSymbol`.

`namespace_divider` is there to divide the namespace from the name. It's allowed to occur only once or never, and there are (in most interpretations) different rules for namespaces and names. E.g., Clojure expects namespaces to approximately be valid Java package names `foo.bar.baz`, so `...` _shouldn't_ be used as a namespace. (That doesn't mean the Clojure reader will reject it.)